src/main/resources/avro/variantmethods.avdl

```diff
@@ -51,6 +51,89 @@ SearchVariantSetsResponse searchVariantSets(
   /** This request maps to the body of `POST /variantsets/search` as JSON. */
   SearchVariantSetsRequest request) throws GAException;
 
+
+/******************  /variantsets/{id}/sequences/search  *********************/
+/**
+This request maps to the body of `POST /variantsets/{id}/sequences/search` as
+JSON.
+*/
+record SearchVariantSetSequencesRequest {
+  /**
+  If nonempty, return `Segment`s describing novel sequences in the `VariantSet`
+  that are joined onto sequences with one of the specified UUIDs.
+  */
+  array<string> parentIds = [];
+
+  /**
+  Specifies the maximum number of results to return in a single page.
+  If unspecified, a system default will be used.
+  */
+  union { null, int } pageSize = null;
+
+  /**
+  The continuation token, which is used to page through large result sets.
+  To get the next page of results, set this parameter to the value of
+  `nextPageToken` from the previous response.
+  */
+  union { null, string } pageToken = null;
+}
+
+/**
+This is the response from `POST /variantsets/{id}/sequences/search` expressed as
+JSON.
+*/
+record SearchVariantSetSequencesResponse {
+  /**
+  The list of `Segment`s for matching sequences from the novel sequence graph
+  for the `VariantSet`.
+  */
+  array<org.ga4gh.models.Segment> segments = [];
+
+  /**
+  The continuation token, which is used to page through large result sets.
+  Provide this value in a subsequent request to return the next page of
+  results. This field will be empty if there aren't any additional results.
+  */
+  union { null, string } nextPageToken = null;
+}
+
+/**
+Gets a list of `Segment` matching the search criteria.
+
+`POST /variantsets/{id}/sequences/search` must accept a JSON version of
+`SearchVariantSetSegmentsRequest` as the post body and will return a JSON
+version of `SearchVariantSetSegmentsResponse`.
+*/
```

Comment:
There's a mismatch between the docs and the definition here. Did you previously have `Segment` rather than `Sequence`?